docscan/sandbox/request/check/third_party_identity_check.go

```diff
@@ -0,0 +1,41 @@
+package check
+
+import "github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/check/report"
+
+// ThirdPartyIdentityCheck defines a sandbox check with a third party credit reporting agency
+type ThirdPartyIdentityCheck struct {
+	*check
+}
+
+// ThirdPartyIdentityCheckBuilder builds a ThirdPartyIdentityCheck
+type ThirdPartyIdentityCheckBuilder struct {
+	checkBuilder
+}
+
+func (c *ThirdPartyIdentityCheck) Report() checkReport {
```

Comment:
This is an exported method with an unexported type. I presume we want to export `checkReport` to solve this?